moveit_ros/hybrid_planning/local_planner/local_constraint_solver_plugins/src/forward_trajectory.cpp

```diff
@@ -43,6 +43,7 @@ const rclcpp::Logger LOGGER = rclcpp::get_logger("local_planner_component");
 // If stuck for this many iterations or more, abort the local planning action
 constexpr size_t STUCK_ITERATIONS_THRESHOLD = 5;
 constexpr double STUCK_THRESHOLD_RAD = 1e-4;  // L1-norm sum across all joints
+constexpr double COLLISION_THRESHOLD = 0.01;  // Stop if closer than this [meters]
```

Comment:
it's not :+1: but due to not having a client to actively fund this, it will probably have to stay that way for now  :(

Feel free to open a PR, of course.